### PR TITLE
Added support for srcset and sizes attributes on the rollup branch

### PR DESCRIPTION
**Pull request by dscafati**

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -37,13 +34,13 @@ <h2>
            Full window, contain, click the numbers to open:
         </h2>
         <div id="example0">
-            <a class="chocolat-image" href="demo-images/5.jpg" title="HO">
+            <a class="chocolat-image" href="demo/demo-images/5.jpg" title="HO">
```
Comment (nicolas-t) [inline demo/index.html]: Why did you change these paths ? 
demo/index.html is meant to be openned directly in the browser without running a http server, like this : 
`file://path/to/demo/index.html`

Comment (dscafati) [reply demo/index.html]: They weren't working for me when I started a new webserver at the root folder of the project. I reverted this
